README.md

```diff
@@ -32,8 +34,8 @@ Check the complete examples in the [examples/](./examples) folder).
 ```go
 func main() {
 	// Defining init, middle and terminal nodes that wrap some functions
```

Comment:
Defining `Start`, ...

Reply:
Search for `Init` through the rest of the code/comments and make the appropriate changes to `Start`.